### PR TITLE
typos

### PR DESCRIPTION
**Pull request by psi29a**
A running log of fixes that I want travis-ci...

### CODE REVIEW

File: src/wm_error.c [hunk 1]
```diff
@@ -113,7 +113,7 @@ void _WM_ERROR(const char * func, unsigned int lne, int wmerno,
         "Unable to load",
         "Unable to open",
         "Unable to read",
-        "Invalid or Unsuported file format",
+        "Invalid or Unsupported file format",
```
Comment (psi29a) [inline src/wm_error.c]: @sezero and @chrisisonwildcode Why is this in wildmidi twice?

I mean the following function:
static const char *errors[WM_ERR_MAX+1]

**Comment by sezero** [comment]
Looks like an oversight to me: The second instance in _WM_ERROR() can be removed. Also, the first entry in the global instance should read "No error" instead of "System Error".

**Comment by sezero** [comment]
These changes seem good for merging to me.

**Comment by psi29a** [comment]
Thanks @sezero

**Pull request merged**